src/ForeignKeyEditorDelegate.cpp

```diff
@@ -29,7 +29,7 @@ class ForeignKeyEditor : public QWidget
         layout->addWidget(clauseEdit);
         layout->addWidget(m_btnReset);
         layout->setSpacing(0);
-        layout->setMargin(0);
+        // layout->setMargin(0);
```

Comment:
In my opinion, if this is not really needed, it should be deleted. Otherwise, we need a comment about what is missing and when we can recover this code.